src/main/java/com/fabriccommunity/thehallow/registry/HallowedEntities.java

```diff
@@ -35,15 +28,21 @@
 	public static final EntityType<RestlessCactusEntity> RESTLESS_CACTUS = register("restless_cactus", FabricEntityTypeBuilder.create(EntityCategory.MISC, RestlessCactusEntity::new).size(EntityDimensions.changing(0.9F, 1.0F)).build());
 	public static final EntityType<MummyEntity> MUMMY = register("mummy", FabricEntityTypeBuilder.create(EntityCategory.MONSTER, MummyEntity::new).size(EntityDimensions.fixed(0.7F, 2.3F)).build());
 	public static final EntityType<CultistEntity> CULTIST = register("cultist", FabricEntityTypeBuilder.create(EntityCategory.MONSTER, CultistEntity::new).size(EntityDimensions.fixed(0.6F, 1.95F)).build());
-	
+	public static final EntityType<ShotgunProjectileEntity> SHOTGUN_PROJECTILE = register(
+		"shotgun_projectile",
+		FabricEntityTypeBuilder.<ShotgunProjectileEntity>create(
```

Comment:
Unnecessary generic declaration

Reply:
I get a compiler error: `error: incompatible types: cannot infer type-variable(s) T` so I have to add that to make it work.

Reply:
You should get the error when you first type it, but after you typed `			ShotgunProjectileEntity::new` the error should go away.

Reply:
I removed the `<ShotgunProjectileEntity>` before `create` and still got a compiler error.

Reply:
Keep it then. Doesn't cause troubles when you leave it there.